### ISSUE TITLE
psycopg2.OperationalError: could not connect to server

### ISSUE THREAD

**Issue by bigmillz** [OCA/OpenUpgrade#695]
Impacted versions:
10.0
Steps to reproduce:
1. Run command `python migrate.py --config=/home/pmiller/Downloads/openerp-server.conf --database=/home/pmiller/Downloads/amzt-odoo-1_2016-12-09_21-26-54.zip --run-migrations=10.0`
Current behavior:
-See below output snippet
Expected behavior:
-Migration of database
Video/Screenshot link (optional):

Impacted versions:
10.0
Steps to reproduce:
1. Run command `python migrate.py --config=/home/user/Downloads/openerp-server.conf --database=/home/user/Downloads/db-odoo-1_2016-12-09_21-26-54.zip --run-migrations=10.0`

Current behavior:
-See below output snippet
Expected behavior:
-Migration of database
Video/Screenshot link (optional):

```
python migrate.py --config=/home/user/Downloads/openerp-server.conf --database=/home/user/Downloads/db-odoo-1_2016-12-09_21-26-54.zip --run-migrations=10.0
linking server/addons to /var/tmp/openupgrade/10.0/addons
getting git://github.com/OpenUpgrade/OpenUpgrade.git
Cloning into '/var/tmp/openupgrade/10.0/server'...
remote: Counting objects: 23005, done.
remote: Compressing objects: 100% (18956/18956), done.
remote: Total 23005 (delta 6116), reused 12196 (delta 3146), pack-reused 0
Receiving objects: 100% (23005/23005), 113.89 MiB | 691.00 KiB/s, done.
Resolving deltas: 100% (6116/6116), done.
Checking connectivity... done.
Checking out files: 100% (20487/20487), done.
Cloning into '/var/tmp/openupgrade/openupgradelib'...
remote: Counting objects: 36, done.
remote: Compressing objects: 100% (30/30), done.
remote: Total 36 (delta 0), reused 24 (delta 0), pack-reused 0
Unpacking objects: 100% (36/36), done.
Checking connectivity... done.
copying database /home/user/Downloads/db-odoo-1_2016-12-09_21-26-54.zip to /home/user/Downloads/db-odoo-1_2016-12-09_21-26-54.zip_migrated...
Traceback (most recent call last):
  File "migrate.py", line 377, in <module>
    db_name = copy_database(conn_parms)
  File "migrate.py", line 25, in copy_database
    conn = psycopg2.connect(**conn_parms)
  File "/usr/lib/python2.7/dist-packages/psycopg2/__init__.py", line 164, in connect
    conn = _connect(dsn, connection_factory=connection_factory, async=async)
psycopg2.OperationalError: could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
```

**Comment by pedrobaeza** [comment]
This is something with your own local installation of Postgres, that is not running on port 5432.

**Comment by HammadShareef** [comment]
@pedrobaeza sir what does means ? i m facing same kind of issue...
Thank in advanced

**Comment by pedrobaeza** [comment]
As said, the problem is not with OpenUpgrade, but with your local installation of Postgres. This is a sysadmin question.

**Comment by kathpalrk** [comment]
Hi @pedrobaeza  

I am also facing an same issue. 


 conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"? - - -
^C2019-11-08 05:47:49,305 439 INFO ? odoo.service.server: Initi


I am using the diff postgresql port which I've already definded under odoo.conf file, but its still not able to communicate. 

I've tried to connect the Postgresql from CLI which is working but its not allowing me to connect the same from odoo web level. Is there any other configuration where we have to mentioned the postgres port and host information apart from conf file.
